### ISSUE TITLE
Verification popup is not closing when I validate my account in an other tab

### ISSUE THREAD

**Issue by BenoitTallandier** [linagora/Twake#1253]
**Describe the bug**
When I signup, I see the email verification popup on Twake. When I verify my email, with an other browser, the popup is still opened in the previous tab.

**To Reproduce**
1. Signup
2. Verify your email
3. Come back on the first Twake tab
4. See the popup

**Screenshots**

https://user-images.githubusercontent.com/22451567/117298514-bd960980-ae77-11eb-8af4-fa77312a51f4.mp4

**Comment by RomaricMourgues** [comment]
Yes it is designed like this, we didn't wanted to implement a boolean in database for that while we are migrating everything to node. Do you think we can keep it like this for now ?

**Comment by BenoitTallandier** [comment]
Yes, it's minor. We shouldn't do it now 👍

**Comment by RomaricMourgues** [comment]
The user realtime is implemented but it seems we don't get an event from console webhooks right now, to be investigated.
